src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -314,6 +314,12 @@ class SettingsMenu extends Menu {
             gearButton.setAttribute('aria-expanded', true);
         }
 
+        if (document.querySelector('.jw-breakpoint-0') || document.querySelector('.jw-breakpoint-1')) {
```

Comment:
we should probably use the query selector for the player element here as it's possible to have multiple players on a page. I'm not sure how exactly you get the player element, i will take a look when I get a chance.